docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,74 @@
+### Description
+
+Upon running the <docker-compose -f docker-compose.yml -f docker-compose.override.yml up> all the containers are not started mentioned in the docker-compose file. Some containers throw certain errors mentioned below
+
+### To Start/Run the app
+
+◦	docker-compose up -d
+◦	docker-compose stop ibf-dashboard
+◦	cd interfaces/IBF-dashboard
+◦	npm start
+
+### Expected behaviour
+All containers mentioned in the docker-compose file /IBF-system/docker-compose.yml be up and running.
+Application should load and respond properly on the localhost:4200.
+
+### Desktop (please complete the following information):
+
+OS: MAC OS
+Browser : chrome
+Version : 92.*
+
+### Workaround
+    ◦	If the containers ibf-system_ibf-pipeline , ibf-system_ibf-api-service do not start.
+    ◦	If the container do not start automatically via the 'docker compose up' command. The workaround for this would be, manually start the container and run this command.
+
+ docker-compose -f docker-compose.yml -f docker-compose.override.yml up
+
+### After step2 of readme if there is any problem to run the app 
+You can try below steps:
+    ◦	throw away "IBF-app" schema of your local database : drop schema "IBF-app" cascade;
```

Comment:
Recreating schema would only be a solution if there's something wrong with your existing setup. So this would never happen on initial first-time set up I guess.. Anyway, fine to leave this in here I guess.